src/internal/test/data-testid-helpers.ts

```diff
@@ -0,0 +1,3 @@
+export const queryByTestId = <T extends Element>(container: HTMLElement, testId: string): T | null => {
+  return container.querySelector<T>(`[data-testid=${testId}]`);
```

Comment:
Probably doesn't matter, but can't hurt just in case.

```suggestion
  return container.querySelector<T>(`[data-testid="${testId}"]`);
```

Reply:
Done, thanks for that!